buildSrc/src/main/kotlin/DependencyVersions.kt

```diff
@@ -0,0 +1,29 @@
+val versions = mapOf(
```

Comment:
what people usually do for this is something like:

```kotlin
object DependencyVersions {
  val checkerQual = "3.8.0"
  val geantyref = "1.3.11
}
```

which gives you some extra type-safety and IDE support

Reply:
I also thought about some extension functions to bring in dependencies like DependencyHandlerScope.checkerQual() but maybe that is going too far

Reply:
I've done that sometimes, but only really for dependencies that need to exist in multiple configurations at once -- often AP + compileOnlyApi annotation lib combinations